### ISSUE TITLE
Create a middleware loader

### ISSUE THREAD

**Issue by IanSeyler** [ReturnInfinity/BareMetal-kernel#19]
The kernel needs code for loading the middleware from disk/memory to the appropriate area and start it.

**Comment by tay10r** [comment]
Isn't this fixed with the Alloy project?

**Comment by IanSeyler** [comment]
Yes. I just haven't closed it yet since Alloy isn't functional just yet. Hopefully will be closing this soon.

**Comment by IanSeyler** [comment]
In its current state Alloy is able to do this.
